data/json/monsters/insect_spider.json

```
@@ -2057,8 +2057,39 @@
     "death_function": { "message": "The %s's body leaks acid.", "effect": { "id": "death_acid", "hit_self": true } },
     "harvest": "arachnid_acid",
     "dissect": "dissect_insect_sample_single",
+    "upgrades": { "half_life": 42, "into": "mon_ant_acid_soldier_mega" },
     "flags": [ "ACIDPROOF", "SHORTACIDTRAIL", "CLIMBS", "HEARS", "POISON", "SEES", "SMELLS", "PATH_AVOID_FIRE", "PATH_AVOID_FALL" ]
   },
+  {
+    "id": "mon_ant_acid_soldier_mega",
+    "type": "MONSTER",
+    "name": { "str": "super soldier acid ant" },
+    "description": "A cow-sized, woolly brown ant.  Corrosive liquid seeping from her huge mandibles.  Given the relative strength of ants to their body size, you don't doubt that this one could shove cars around or rip through solid walls.",
+    "copy-from": "mon_ant_acid_soldier",
+    "proportional": { "hp": 5, "speed": 0.85, "dodge": 0.5, "vision_day": 2 },
+    "volume": "625 L",
+    "weight": "815 kg",
+    "melee_dice": 2,
+    "grab_strength": 3,
+    "armor_bash": 15,
+    "armor_cut": 24,
+    "armor_bullet": 15,
+    "armor_elec": 4,
+    "bleed_rate": 60,
+    "special_attacks": [
+      { "id": "smash", "throw_strength": 48, "cooldown": 30 },
+      { "id": "leg_sweep" },
+      {
+        "type": "bite",
+        "cooldown": 5,
```

```suggestion
        "cooldown": 5,
        "forbidden_effects_any": [ "maimed_mandible" ],
```
Weakpoint disabling effects.